tests/modules/saml/lib/Auth/Source/Auth_Source_SP_Test.php

```diff
@@ -121,6 +121,9 @@ protected function setUp()
                 ),
             ),
         );
+
+        // Modules makes use of HTTP for getting the base url which attempts to load a config.php
+        putenv('SIMPLESAMLPHP_CONFIG_DIR=tests/config/no-options');
```

Comment:
I'll change to `SimpleSAML_Configuration::loadFromArray(array(), '[ARRAY]', 'simplesaml');`